### PR TITLE
Add tests to increase test coverage for Coin.cpp

### PR DESCRIPTION
**Pull request by johnnynanjiang**
## Description
Increase test coverage for Coin.cpp

### CODE REVIEW

File: tests/CoinTests.cpp [hunk 1]
```diff
@@ -186,13 +186,65 @@ TEST(Coin, ValidateAddressDGB){
     EXPECT_FALSE(validateAddress(TWCoinTypeDigiByte, "SUngTA1vaC2E62mbnc81Mdos3TcvZHwsVx"));
 }
 
+TEST(Coin, validateAddressIocoin) {
+    EXPECT_TRUE(validateAddress(TWCoinTypeIocoin, "ik6V522aupkj3kMtJZi6eDw6v3eLgMbXY8"));
```
Comment (vikmeup) [inline tests/CoinTests.cpp]: Did you consider adding a new file: `CoinAddressValidationTests.cpp` ? just

**Comment by hewigovens** [comment]
hmm, seems codecov didn't run?

File: tests/CoinTests.cpp (hunk 1 above)
Comment (johnnynanjiang) [reply tests/CoinTests.cpp]: Done in 3351155e1177f17cc56d62cfeacfcf0ba26a70e2

**Comment by vikmeup** [comment]
@johnnynanjiang might worth adding this step to https://github.com/TrustWallet/developer/blob/master/wallet-core/newblockchain.md#blockchain-checklist too, to implement tests in both of the files

**Comment by johnnynanjiang** [comment]
> @johnnynanjiang might worth adding this step to https://github.com/TrustWallet/developer/blob/master/wallet-core/newblockchain.md#blockchain-checklist too, to implement tests in both of the files
> 
> >  GitHub**[TrustWallet/developer](https://github.com/TrustWallet/developer)**Contribute to TrustWallet/developer development by creating an account on GitHub.

Raised a PR https://github.com/TrustWallet/developer/pull/5

**Comment by johnnynanjiang** [comment]
> hmm, seems codecov didn't run?

codecov is back automagically

**Pull request merged**